src/AmbientSounds.Uwp/App.xaml.cs

```diff
@@ -108,17 +108,25 @@ public static IServiceProvider Services
         protected override async void OnLaunched(LaunchActivatedEventArgs e)
         {
             await ActivateAsync(e.PrelaunchActivated);
+            if(e.Kind == ActivationKind.Protocol)
+            {
+                HandleProtocolLaunch((ProtocolActivatedEventArgs)(IActivatedEventArgs)e);
```

Comment:
maybe modify the if condition to include `e is ProtocolActivatedEvenetArgs args`? then we can pass in args to `HandleProtocolLaunch`

Reply:
Yes, but seems we cannot typecast `LaunchActivatedEventArgs` directly using **is**

Reply:
oh right! Maybe let's do `e is IActivatedEventArgs eventArgs && eventArgs is ProtocolActivatedEvenetArgs args`?

Reply:
Hah, Yes! Code changed.